docs/concepts/policy/pod-security-policy.md

```diff
@@ -1,97 +1,359 @@
-of the range as the default. Validates against the configured range.
-- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
-`runAsUser` or have the `USER` directive defined in the image. No default
-provided.
-- *RunAsAny* - No default provided. Allows any `runAsUser` to be specified.
 
-### SELinux
+## Enabling Pod Security Policies
 
-- *MustRunAs* - Requires `seLinuxOptions` to be configured if not using
-pre-allocated values. Uses `seLinuxOptions` as the default. Validates against
-`seLinuxOptions`.
-- *RunAsAny* - No default provided. Allows any `seLinuxOptions` to be
-specified.
+Pod security policy control is implemented as an optional [admission
+controller](/docs/admin/admission-controllers/#podsecuritypolicy). PodSecurityPolicies
+are enforced by [enabling the admission
+controller](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in),
+but doing so without authorizing any policies **will prevent any pods from being
+created** in the cluster.
 
-### SupplementalGroups
+Since the pod security policy API (`extensions/v1beta1/podsecuritypolicy`) is
+enabled independently of the admission controller, for existing clusters it is
+recommended that policies are added and authorized before enabling the admission
+controller.
 
-- *MustRunAs* - Requires at least one range to be specified. Uses the
-minimum value of the first range as the default. Validates against all ranges.
-- *RunAsAny* - No default provided. Allows any `supplementalGroups` to be
-specified.
+## Authorizing Policies
 
-### FSGroup
+Once a PodSecurityPolicy resource has been created, it does nothing. In order to
+use it, the requesting user or target pod's [service
+account](/docs/tasks/configure-pod-container/configure-service-account/) must be
+authorized to use the policy, by allowing the `use` verb on the policy.
 
-- *MustRunAs* - Requires at least one range to be specified. Uses the
-minimum value of the first range as the default. Validates against the
-first ID in the first range.
-- *RunAsAny* - No default provided. Allows any `fsGroup` ID to be specified.
+Most Kubernetes pods are not created directly by user. Instead, they are
+typically created indirectly as part of a
+[Deployment](/docs/concepts/workloads/controllers/deployment/),
+[ReplicaSet](/docs/concepts/workloads/controllers/replicaset/), or other
+templated controller via the controller manager. Granting the controller access
+to the policy would grant access for *all* pods created by that the controller,
+so the preferable method for authorizing policies is to grant access to the
+pod's service account (see [example](#run-another-pod)).
+
+### Via RBAC
+
+[RBAC](/docs/admin/authorization/rbac/) is a standard Kubernetes authorization
+mode, and can easily be used to authorize use of policies.
+
+First, a `Role` or `ClusterRole` needs to grant access to `use` the desired
+policies. The rules to grant access look like this:
+
+```yaml
```

Comment:
Better make this snippet a complete manifest for users, i.e. add apiVersion, kind, metadata.

Reply:
Done.